python3/vim_pandoc/command.py

```diff
@@ -173,11 +181,11 @@ def execute(self, should_open):
             # for vim versions with clientserver support
             elif vim.eval("has('clientserver')") == '1' and \
                  vim.eval("v:servername") != "" and \
-                 vim.eval("executable('python')") == '1':
+                 self.python_executable() is not None:
```

Comment:
Maybe safe the outcome of this test here for use later without re-probing the filesystem.

Reply:
How about this?

Reply:
I think that's better, thanks!